Source/WebCore/loader/appcache/ApplicationCacheStorage.cpp

```diff
@@ -1269,7 +1269,7 @@ void ApplicationCacheStorage::deleteTables()
 bool ApplicationCacheStorage::shouldStoreResourceAsFlatFile(ApplicationCacheResource* resource)
 {
     auto& type = resource->response().mimeType();
-    return startsWithLettersIgnoringASCIICase(type, "audio/"_s) || startsWithLettersIgnoringASCIICase(type, "video/"_s);
+    return startsWithLettersIgnoringASCIICase(type.string(), "audio/"_s) || startsWithLettersIgnoringASCIICase(type.string(), "video/"_s);
```

Comment:
Can this ambiguity be solved with more overloading? Not so great to be required to touch a call site like this one that is unambiguous to a human being.

Reply:
Good point, will fix.